source/acquaman/AMXRFScanController.cpp

```diff
@@ -97,7 +97,7 @@ void AMXRFScanController::onDetectorAcquisitionFinished()
 		}
 	}
 
-	else {
+	else if (!detector_->inputCountSources().isEmpty() && !detector_->outputCountSources().isEmpty()){
```

Comment:
I think we don't need `!detector_->inputCountSources().isEmpty()`, since the `detector_->inputCountSources()` is for sure not empty in the else branch